include/extractor/guidance/toolkit.hpp

```diff
@@ -369,29 +370,32 @@ inline bool requiresNameAnnounced(const std::string &from, const std::string &to
     // references are contained in one another
     auto refs_are_empty = from_ref.empty() && to_ref.empty();
     auto ref_is_contained =
-        !from_ref.empty() && !to_ref.empty() &&
+        from_ref.empty() || to_ref.empty() ||
         (from_ref.find(to_ref) != std::string::npos || to_ref.find(from_ref) != std::string::npos);
     auto ref_is_removed = !from_ref.empty() && to_ref.empty();
 
-    auto obvious_change = ref_is_contained || names_are_equal ||
-                          (names_are_empty && refs_are_empty) || name_is_removed || ref_is_removed;
+    auto obvious_change = (names_are_empty && refs_are_empty) ||
+                          (names_are_equal && ref_is_contained) ||
+                          (names_are_equal && refs_are_empty) || name_is_removed || ref_is_removed;
 
     return !obvious_change;
 }
 
-inline int getPriority( const FunctionalRoadClass road_class )
+inline int getPriority(const FunctionalRoadClass road_class)
 {
-    //The road priorities indicate which roads can bee seen as more or less equal.
-    //They are used in Fork-Discovery. Possibly should be moved to profiles post v5?
-    //A fork can happen between road types that are at most 1 priority apart from each other
-    const constexpr int road_priority[] = {10, 0, 10, 2, 10, 4, 10, 6, 10, 8, 10, 11, 10, 12, 10, 14};
+    // The road priorities indicate which roads can bee seen as more or less equal.
+    // They are used in Fork-Discovery. Possibly should be moved to profiles post v5?
+    // A fork can happen between road types that are at most 1 priority apart from each other
+    const constexpr int road_priority[] = {
+        10, 0, 10, 2, 10, 4, 10, 6, 10, 8, 10, 11, 10, 12, 10, 14};
```

Comment:
For the record

    $ clang-format --version
    clang-format version 3.8.0 (tags/RELEASE_380/final)

    $ clang-format -style=file -i ../include/extractor/guidance/toolkit.hpp